bpo-43848: explain optional argument mtime in gzip.py. (#25410)
python/cpython: Doc/library/gzip.rst

   compression, and ``9`` is slowest and produces the most compression. ``0``
   is no compression. The default is ``9``.

   The optional *mtime* argument is the timestamp requested by gzip. The time

>    is in Unix format, i.e., seconds since 00:00:00 GMT, Jan.  1, 1970.

Reviewer: ```suggestion
   is in Unix format, i.e., seconds since 00:00:00 UTC, January 1, 1970.
```
Avoid abbreviations, and UTC is both more recognizable and more precise.

>    If mtime is omitted or None, the current time is used. Use mtime = 0

Reviewer: ```suggestion
   If *mtime* is omitted or None, the current time is used. Use *mtime* = 0
```

   .. attribute:: mtime

>       When decompressing, this attribute is set to last timestamp in the most
>       recently read header.  It is an integer, holding the number of seconds
>       since the Unix epoch (00:00:00 GMT, Jan. 1, 1970).

Reviewer: ```suggestion
      When decompressing, this attribute is set to the last timestamp in the most
      recently read header.  It is an integer, holding the number of seconds
      since the Unix epoch (00:00:00 UTC, January 1, 1970).
```
